### PR TITLE
Completed new script to generate sample data

### PR DESCRIPTION
**Pull request by F-Ting**

### CODE REVIEW

File: test/back_end/api_get_test.spec.js [hunk 1]
```diff
@@ -5,9 +5,9 @@ chai.use(chaiHttp);
 const expect = chai.expect;
 
 describe("Test GET requests to API", function() {
-    it("should return user info for prof1", async function() {
+    it("should return user info for Joshua Cook", async function() {
         let fetched = await fetch(
-            "http://localhost:3000/get_info.php?what=user_info&user_id=prof1"
+            "http://localhost:3000/get_info.php?what=user_info&user_id=cook19"
         ).catch(function(err) {
```
Comment (siefkenj) [inline test/back_end/api_get_test.spec.js]: No mixing `await` and `.fetch()`

Comment (F-Ting) [reply test/back_end/api_get_test.spec.js]: I have to wait for the fetch result to come back before I can do assert statements. Do you mean removing the `.catch`?

Comment (siefkenj) [reply test/back_end/api_get_test.spec.js]: I don't follow. `fetch` returns a promise which you can use `.catch` with. `await` awaits a promise and turns `.catch` into a regular JS error. Thus
```
fetch(...).catch(...)
```
is the same as
```
try {
   await fetch(...)
} catch (e) {
   ...
}
```

Comment (F-Ting) [reply test/back_end/api_get_test.spec.js]: That what I meant, you said to remove the `.fetch` which I got me thinking you wanted to not use `await` . I'll update according to that.

**Comment by F-Ting** [comment]
Actually Don't Merge this yet, I'm looking into a missing association link in our data.

**Pull request merged**